microsoft-edge/extensions-chromium/developer-guide/minimize-page-load-time-impact.md

```diff
@@ -0,0 +1,251 @@
+
+1. Type **Evaluate script** and then press **Enter** until the **Performance** tool highlights the performance events that are caused by your content script.  You will know that you have found the right performance event when the **Script**  label in the **Summary** panel shows the name of your content script.
+
+   ![The Evaluate script performance event due to the extension's content script running during the page load](./minimize-page-load-time-impact-images/evaluate-script-event.png)
+
+
+<!-- ------------------------------ -->
+#### Profile your content script using the Edge tracing tool
+
+The Edge tracing tool, available at the `edge://tracing` URL, is a powerful tool that can provide detailed analysis of your extension's performance. In this section, you'll learn how to use the Edge tracing tool to understand the impact of your extension on the page load time. To learn more about the Edge tracing tool, which is based on the **Perfetto** tool, see [Perfetto UI](https://perfetto.dev/docs/visualization/perfetto-ui).
+
+1. To open the Edge tracing tool, open a new tab or window, and go to `edge://tracing`. The tracing UI opens.
+
+1. To start a new trace, in the top left corner of the tool, click **Record**. The **Record a new trace** dialog box appears.
+
+1. In the **Record a new trace** dialog box, select the **Manually select settings** option. The list of categories appears.
+
+1. To capture detailed information about your extension's content script compilation and execution, select the following categories:
+
+   * **extensions**
+   * **v8**
+   * **devtools**
+   * **devtools.timeline**
+
+1. Click **Record**. The Edge tracing tool starts recording the trace.
+
+1. Open a new tab and load a webpage that your extension affects. The tracing tool collects data about the performance impact of your extension on the webpage.
+
+1. Open the tab where the Edge tracing tool is running, and then click **Stop**. The new trace information appears in the tool.
+
+Traces recorded by the Edge tracing tool provide a lot of information about the browser. To filter the information to only what's relevant to the webpage that your extension impacted, use the following steps:
+
+1. Press **Shift+Esc** to open the **Browser Task Manager** dialog box.
+
+1. In the **Browser Task Manager** dialog box, search for the tab that corresponds to the webpage that your extension affected, and note the number in the **Process ID** column. Close the dialog box.
+
+1. In the toolbar of the Edge tracing tool, click **Processes**, and then select the checkbox that corresponds to the process ID you noted. Clear all other checkboxes.
+
+1. In the top right corner of the Edge tracing tool, click the search field, type **ScriptInjection::InjectJS**, and then press **Enter** until an event that corresponds to your extension gets highlighted in the bottom panel.
```

Comment:
```suggestion
1. In the top right corner of the Edge tracing tool, click the **Search** field, type **ScriptInjection::InjectJS**, and then press **Enter** until an event that corresponds to your extension gets highlighted in the bottom panel.
```
check whether theres a UI control label or dimmed text in text box

rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/minimize-page-load-time-impact?branch=pr-en-us-3272

Reply:
no label or tooltip